catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -155,7 +162,6 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
                     }
                   }}
                   layout="fill"
-                  background={background}
```

Comment:
We can see the background color in these instances, so it's better to leave it to the default `transparent`.